core/primitives/transaction_validity.hpp

```diff
@@ -53,123 +54,139 @@ namespace kagome::primitives {
      */
     TransactionLongevity longevity;
 
-    bool operator==(const Valid &rhs) const {
+    /**
+     * @brief A flag indicating if the transaction should be propagated to other
+     * peers. By setting `false` here the transaction will still be considered
+     * for including in blocks that are authored on the current node, but will
+     * never be sent to other peers.
+     */
+    bool propagate;
```

Comment:
Maybe set the default value?

Reply:
Why? This should be set by the user who creates transaction

Reply:
Although, you are right. Clang-tidy requires to set default value